ts/src/bybit.ts

```diff
@@ -1719,9 +1721,16 @@ export default class bybit extends Exchange {
             await this.loadTimeDifference ();
         }
         const promisesUnresolved = [];
-        const fetchMarkets = this.safeList (this.options, 'fetchMarkets', [ 'spot', 'linear', 'inverse' ]);
-        for (let i = 0; i < fetchMarkets.length; i++) {
-            const marketType = fetchMarkets[i];
+        let types = undefined;
+        const fetchMarketsOptions = this.safeDict (this.options, 'fetchMarkets');
+        if (fetchMarketsOptions !== undefined) {
+            types = this.safeList (fetchMarketsOptions, 'types', []);
+        } else {
+            // for backward-compatibility
+            types = this.safeList (this.options, 'fetchMarkets', []);
```

Comment:
@ttodua why you get rid of the defaults, we should always have a default list in case the user removes options completely or smth like that?

Also, in this case `types` does not make much sense (since linear/inverse are the same type) but we will have this problem in a lot of different exchanges